recipes/libgpiod/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import platform
+import re
+
+required_conan_version = ">=1.33.0"
+
+class LibgpiodConan(ConanFile):
+    name = "libgpiod"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/"
+    license = "LGPL-2.1-or-later"
+    description = "C library and tools for interacting with the linux GPIO character device"
+    topics = ("conan", "gpio", "libgpiod", "libgpiodcxx", "linux")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "enable_bindings_cxx": [True, False],
+               "enable_tools": [True, False]}
+    default_options = {"shared": False,
+                       "enable_bindings_cxx": False,
+                       "enable_tools": False}
```

Comment:
Code style suggestion
```suggestion
    options = {
        "shared": [True, False],
        "enable_bindings_cxx": [True, False],
        "enable_tools": [True, False],
    }
    default_options = {
        "shared": False,
        "enable_bindings_cxx": False,
        "enable_tools": False,
    }
```